build/_build/docker/debian.dockerfile

```diff
@@ -0,0 +1,56 @@
+ARG DOTNETSDK_VERSION
+# debian 10 image
+FROM mcr.microsoft.com/dotnet/sdk:$DOTNETSDK_VERSION-buster-slim as base
+# ubuntu image
+# FROM mcr.microsoft.com/dotnet/sdk:$DOTNETSDK_VERSION-focal
+
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --fix-missing \
+        git \
+        wget \
+        curl \
+        cmake \
+        make \
+        llvm \
+        clang \
+        gcc \
+        build-essential \
+        rpm \
+        ruby \
+        ruby-dev \
+        rubygems \
+    && gem install --no-document fpm
+
+
+ENV CXX=clang++
+ENV CC=clang
+
+FROM base as builder
+
+# Copy the build project in and build it
+COPY . /build
+RUN dotnet build /build
+WORKDIR /project
+
+FROM base as tester
+
+# Install .NET Core runtimes using install script
+# FYI there is no arm64 runtime available for .NET Core 2.1, so just install the .NET Core runtime in that case
```

Comment:
I think some of these were meant to say "ASP.NET Core runtime" instead of ".NET Core runtime". (`aspnetcore` vs `dotnet`)
```suggestion
# Install ASP.NET Core runtimes using install script
# There is no arm64 runtime available for .NET Core 2.1, so just install the .NET Core runtime in that case
```